src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: {
+    categories: [],
+    status: 'Under construction',
+  },
+  reducers: {
+    statusCheck: (state) => ({ ...state, status: 'Under construction' }),
+  },
+});
```

Comment:
- Please ensure that in your `src/components/categories.js`, instead of coming soon you should have a `Check Status` button that outputs `under construction` on click 👍🏽

Reply:
Ok